### ISSUE TITLE
Acre2 Babel voice being applied to every player

### ISSUE THREAD

**Issue by PrivateMcMuffin** [IDI-Systems/acre2#564]
**Arma 3 Version:** `1.84.144923` (stable / rc / dev)
**CBA Version:** `3.8.0` (stable / dev + commit hash)
**ACRE2 Version:** `stable` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3
- acre2
```

**Description:**
Other players and myself experience a very deep robotically babel-like effect on our direct/radio voices that is hard to understand. All players are on the same side and same group.
**Steps to reproduce:**
- Add the steps needed to reproduce the issue.
Install CBA/Acre2, host server with any mission file on it. Spawn in with selected role.
**Where did the issue occur?**
- Dedicated server as well as a Self-Hosted Multiplayer 

**Placed Modules:**
reproduced with no placed modules.
I've also tried messing with both acre modules.

**RPT log file:**
https://pastebin.com/PNKGJBhi

**Comment by TheMagnetar** [comment]
I believe that this is  more a babel setup problem in your end than an actual  bug. You  have to setup the babel languages  in every machine. During init,

```
acre2_languages =   [
    ["english", "English"],
    ["russian", "Russian"]
];

{
    _x call acre_api_fnc_babelAddLanguageType;
} foreach acre2_languages;
```

and wherever the unit is local you call the functions `acre_api_fnc_babelSetSpokenLanguages` and `call acre_api_fnc_babelSetSpeakingLanguage`

**Comment by jonpas** [comment]
Are you trying to apply babel, or is ACRE2 without *any* configuration also not work correctly for you?

**Comment by PrivateMcMuffin** [comment]
ACRE2 without any configuration isn't working correctly for me. 
I've gone ahead and Tried with zero Configuration, as well as run a mission with a Init file setup by another user for acre. At the moment I am currently setting up a simple mission with a simple init file like you've mentioned.

(Edit)I've inputted in those few lines into a simple init file, and it actually made it worse if anything. I don't believe it changed anything

**Comment by TheMagnetar** [comment]
Can you post the mission?

**Comment by PrivateMcMuffin** [comment]
I can, I tried it on about 12 different missions, some with mods, some without. I got the same effect on all of them. I'm attaching two mission files I tried, the first is one from a different user that is a large op, it had some mods for it. The other is the simple mission I made to try to get acre to work with CBA and acre.

[missions.zip](https://github.com/IDI-Systems/acre2/files/2268222/missions.zip)
 the mission I made myself is only a spawned set of units, and a init file for babel. Originally there was no init file for it. I received the same effect no matter what configuration of any kind I used

**Comment by TheMagnetar** [comment]
This is because you do not apply the lines above correctly. you have to specify the languages you want to apply the language as an argument to the functions plus you do not call them where the unit is local (it gets executed everywhere). I see that you play the Two-Sierra campaign: if you have a babel problem in there it is for sure a mission configuration problem and I would contact the author of the mission directly.

If you start a mission fresh, make sure no templates/compositions are applied nor any acre modules are applied (I cannot open you mission.sqm from my computer since it is binarised and I do not have access to arma).

I will provide you a fresh mission for you to test.

**Comment by PrivateMcMuffin** [comment]
I appreciate that. I did make a fresh mission a few times in my frustrated two days of trying to understand why it wasn't working. Just threw about 4 units down, launched it up on my dedicated server, and it still had the super deep voices. that's when I started fucking with modules and settings. I've tried the two sierra missions, a few missions from steam. I made about 6 myself all starting with no configuration at all to fucking with some of the files and modules. I even went out to a few community leaders of about three units and asked how they have it set up and what I did wrong.

On a side note, I kinda figured I didn't set up that Init file right. I usually don't mess with those that much, and I don't have much experience with arma 3 scripting in the least bit, spent most of my time with garbage java code.

**Comment by jonpas** [comment]
@TheMagnetar status?

**Comment by jonpas** [comment]
Can't reproduce, closing. Reopen if it reappears.

**Comment by PrivateMcMuffin** [comment]
I seem to be having this issue again. I don't understand why it is happening. I had set acre2 up on a dedicated linux server last week. This problem happened then, but I was able to fix it with the fix illustrated here: https://github.com/IDI-Systems/acre2/issues/413
That seemed to fix it for the last week. Today when I launched my server and attempted to play a stock arma 3 mission, every bodies voices sounded like that were the babel voices. I didn't change anything since the previous play sessions. I went and uninstalled everything on the server and reset it up to no avail.
